green_cli/authenticators/hardware.py

```diff
@@ -0,0 +1,83 @@
+import hwilib.commands
+import wallycore as wally
+
+from green_cli.authenticators import *
+
+class HWIDevice(HardwareDevice):
+
+    @staticmethod
+    def _path_to_string(path: List[int]) -> str:
+        """Return string representation of path for hwi interface
+
+        The gdk passes paths as lists of int, hwi expects strings with '/'s
+        >>> _path_to_string([1, 2, 3])
+        "m/1/2/3"
+        """
+        return '/'.join(['m'] + [str(path_elem) for path_elem in path])
```

Comment:
Subaccount keys are `m/3'/subaccount'/1/pointer` (note dashed 3 and subaccount for private derivation) - this doesn't seem to cover that?